library/Vanilla/Utility/SchemaFilters.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license https://opensource.org/licenses/GPL-2.0 GPL-2.0
+ */
+
+namespace Vanilla\Utility;
+
+use Garden\Schema\ValidationField;
+
+/**
+ * A collection of field filter functions, compatible with Garden Schema.
+ */
+class SchemaFilters {
+
+    /**
+     * Attempt to decode a encoded string value into a more complex type.
+     *
+     * @param mixed $value Raw database value.
+     * @param ValidationField $field An object representing the Garden Schema field.
+     * @return mixed Unpacked attributes field value.
+     */
+    public static function encodeValue($value, ValidationField $field) {
+        if ($value === null || $value === '') {
+            $value = null;
```

Comment:
Why reset a `null` to `null`?

Reply:
I guess there's 2 schools of thought here - either we value a single return more, or not branching our logic more. Over the years, I've moved from the former to the latter. My feeling is this would do better as `if null return null` and then just return the json encoding as a matter of course otherwise. I'm unbothered if you have reason to keep an else here.

Reply:
This was copied over from the `dbencode` function, but modified to have one return value instead of multiple. In general, I prefer that pattern for debuggging (can just break on the one return statement). In this case, the function is so small that I don't think adding another return statement would hinder its readability or debug-ability.